src/shared/filesystemUtilities.ts

```diff
@@ -51,6 +53,23 @@ export const readFileAsString = async (
     return readFile(pathLike, options)
 }
 
+/**
+ * Best-effort delete a folder recursively. Will not throw if it fails.
+ * @param folder The path to the folder to delete
+ */
+export async function tryRemoveFolder(folder?: string) {
+    try {
+        // if null or empty, no issues
+        if (!folder) {
+            getLogger().warn(`No folder passed into tryRemoveFolder: ${folder}`)
+            return
+        }
+        await del(folder, { force: true })
+    } catch (err) {
+        getLogger().warn(err as Error)
```

Comment:
Often useful to see the input that failed:

```suggestion
        getLogger().warn(`tryRemoveFolder: failed to delete directory '%s': %O`, folder, err as Error)
```